### PR TITLE
Nevin about

### PR DESCRIPTION
**Pull request by nevingeorge**
Created the about page.

### CODE REVIEW

File: DSA-Campaign-Uplift-Estimation/src/main/webapp/About/about.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    <div class="content">
+        <h3>Project Overview</h3>
+        <p>This project aims to address two main problems with the current Dynamic Search Ads framework. The first
+            involves the lack of transparency associated with automated products and the limited predictability of DSA and 
+            machine learning in general. Users have no guarantee that using DSA will produce the results they intend,
+            creating some risk for users attempting to transition over from their traditional keyword campaigns
+            to DSA.</p>
+        <p>The second issue involves the sometimes unnecessary complexity of DSA. Currently over 40% of created DSA
+            campaigns have a serving rate of 0, meaning that the campaign generates no clicks and therefore brings 
+            no revenue. This is due to some misconfiguration while setting up the DSA campaigns, and since this happens
```
Comment (avenkat-google) [inline DSA-Campaign-Uplift-Estimation/src/main/webapp/About/about.html:37]: I would leave this metric out of the about page, as these are internal metrics that we don't want to share outside.

File: DSA-Campaign-Uplift-Estimation/src/main/webapp/About/about.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        <p>This project aims to address both of these issues by allowing advertisers to run test DSA campaigns with
+            easy-to-use UIs, so they can estimate the uplift DSA can bring to their keyword campaigns. Users can
+            experiment with different DSA campaign settings when creating their DSA campaigns, and the results of the
+            campaigns are displayed on the home and compare pages in clear, efficient charts and graphs. By allowing 
+            users to clearly see the value DSA can bring to their ad campaigns, this project hopes to convince more 
+            users to incorporate DSA into their ad campaigns.</p>
+        <p>This project is a model of what DSA could eventually implement. Many changes would have to be made for this
+            project to be used in production, such as a more accurate estimation feature, a cleaner, simpler UI, and 
+            auto-generated campaign settings using AI. This project would satisfy a very big need of advertisers today,
+            and if implemented, Google PMs predict it could potentially open a $750M market.</p>
```
Comment (avenkat-google) [inline DSA-Campaign-Uplift-Estimation/src/main/webapp/About/about.html:48]: Same here

File: DSA-Campaign-Uplift-Estimation/src/main/webapp/About/about.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        <p>The compare page allows users to easily compare the results of their different DSA campaigns. Users can select
+            any number of their created DSA campaigns to compare, and the uplift statistics will be displayed along with 
+            the inputted campaign settings and generated search query reports.</p>
+        <h3>Settings Explanations</h3>
+        <ul>
+            <li>Start/End Dates (Optional) - used to calculate the campaign duration for uplift estimations. The default values
+                 for the start and end dates are the present date and one year in the future, respectively.</li>
+            <li>Manual CPC - how much to spend for every ad click.</li>
+            <li>Daily Budget - maximum amount to spend in a day.</li>
+            <li>Locations - locations to target. Currently only the US and US states are supported. Ex: display ads to users
```
Comment (avenkat-google) [inline DSA-Campaign-Uplift-Estimation/src/main/webapp/About/about.html:67]: maybe rephrase to: ads from the campaign will target users at the given locations. Currently only the US and US states are supported.

File: DSA-Campaign-Uplift-Estimation/src/main/webapp/About/about.html (hunk 1 above)
Comment (nevingeorge) [reply DSA-Campaign-Uplift-Estimation/src/main/webapp/About/about.html]: done

File: DSA-Campaign-Uplift-Estimation/src/main/webapp/About/about.html (hunk 2 above)
Comment (nevingeorge) [reply DSA-Campaign-Uplift-Estimation/src/main/webapp/About/about.html]: done

File: DSA-Campaign-Uplift-Estimation/src/main/webapp/About/about.html (hunk 3 above)
Comment (nevingeorge) [reply DSA-Campaign-Uplift-Estimation/src/main/webapp/About/about.html]: done
